crowd_anki/config.json

```diff
@@ -1,4 +1,3 @@
 {
-  "automated_snapshot": false,
```

Comment:
why?

Reply:
This file is not needed to set a user default anymore. The default values are set on each property in the config class. Adding further properties there would only add a requirement to add them here (and adding them here would still do nothing, as nothing reads this).